packages/registrar_client/lib/src/farms.dart

```diff
@@ -9,15 +9,16 @@ class Farms {
 
   Farms(this._client);
 
-  Future<int> create(String farmName, bool dedicated, String stellarAddress,
-      int twinID) async {
+  Future<int> create(
+      String farmName, bool dedicated, String stellarAddress) async {
+    final twinId = _client.twinId!;
```

Comment:
Please use a null check before accessing the twinId. 
I id not set you will get `Unhandled exception:
Null check operator used on a null value`